app/views/handlers/feedback.py

```diff
@@ -58,7 +63,28 @@ def get_page_title(self):
         return self.PAGE_TITLE
 
     def handle_post(self):
-        self._session_store.session_data.feedback_count += 1
+        session_data = self._session_store.session_data
+        session_data.feedback_count += 1
+
+        metadata = {
+            "feedback_count": session_data.feedback_count,
+            "form_type": self._schema.form_type,
+            "language_code": session_data.language_code,
+            "object_key": str(uuid4()),
+            "region_code": self._schema.region_code,
+            "tx_id": session_data.tx_id,
+        }
+        data = {
+            "feedback_text": self.form.data["feedback-type"],
+            "feedback_topic": self.form.data["feedback-text"],
```

Comment:
```suggestion
            "feedback_topic": self.form.data["feedback-type"],
            "feedback_text": self.form.data["feedback-text"],
```

Also, if we're using `topic` rather than `type` here, then it should be renamed to `topic` in the form to be consistent.